getting_started/data_backup.md

```diff
@@ -0,0 +1,220 @@
+
+> Remember to replace `user` with your username
+
+
+| Action                    | Command                                             |
+| ------------------------- | --------------------------------------------------- |
+| Copy a File from Arjuna   | `rsync user@arjuna.psc.edu:path/to/file path/to/dst`  |
+| Copy a File to Arjuna     | `rsync path/to/file user@arjuna.psc.edu:path/to/dst`  |
+| Copy a Folder from Arjuna | `rsync -r user@arjuna.psc.edu:path/to/folder path/to` |
+| Copy a Folder to Arjuna   | `rsync -r path/to/folder user@arjuna.psc.edu:path/to` |
+
+Unlike `scp`, `rsync` checks files sizes and modification times to only transfer files that have changed. This is helpfully for several reasons:
+
+1. Resume interrupted transfers *without* repeating the entire transfer.
+2. Reduce transfer times by *skipping* already transferred files.
+
+To read the manual page for `rsync`, type `man rsync` into a terminal.
+
+## Transfers to Cloud Storage: `rclone`
+
+[`rclone`](https://rclone.org) can be used to transfer files to cloud storage providers (Google Drive, Box, Amazon S3, SFTP, [and more](https://rclone.org/overview/)). Unlike `scp` and `rsync`, `rclone` can be run from your local machine or arjuna but require additional configuration to operate.
+
+### Installation
+
+1. Download from the [rclone website](https://rclone.org/downloads/).
+2. Install with [homebrew](https://brew.sh/): `brew install rclone`
+
+### Configuring `rclone` Remotes
+
+`rclone` requires configuration of remote destinations ("Remotes") before it can sync files to them. To configures `rclone` use [`rclone config`](https://rclone.org/docs/#configure) to launch an interactive configuration tool. In general adding a new remote is as simple as:
+
+1. Press `n` for a new config.
+2. Enter a name for the new remote (e.g. "box_cmu" or "gdrive_cmu").
+3. Choosing the type of remote. (e.g. "Google Drive",  "Box", "S3", etc.).
+4. Set remote-specific options.
+5. Authenticate with the remote.
+
+We have provided additional instructions for [Google Drive](#google-drive) and [Box](#box), but official documentation can be found on the [rclone website](https://rclone.org/overview/), by clicking "Storage Systems" and choosing the appropriate storage system.
+
+#### Google Drive
+
+After completing [steps 1 - 3 above](#configuring-rclone-remotes): you will be prompted to enter a [`client_id`](https://rclone.org/drive/#making-your-own-client-id)
+
+1. When prompted from a [`client_id`](https://rclone.org/drive/#making-your-own-client-id), flow the instruction on the linked page.
+    - *You must use a non-CMU account* due to institutional restrictions.
+    - You *can* use this client id to sync with your Andrew GDrive account.
+    - You can skip this step, but performance may be limited.
+
+2. Paste your `client_id` and then `secret` when prompted by `rclone configure`.
+3. The default options are fine until reaching:
```

Comment:
Change to "The default options are fine until reaching "Use auto config?"" or something to make it flow a little better